### PR TITLE
docs: node 14 requirement

### PR DESCRIPTION
**Pull request by AndrewLeedham**
- Adds documentation about the Node 14 requirement in both the readme and quick-start guide.
- Updates the engines field to be `>=14`.
- Adds an `.nvmrc` file so contributors of [nvm](https://github.com/nvm-sh/nvm) are switched automatically to the right version of node.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ All you need is an empty directory to get started! But if you prefer a starter p
 
 First, install Slinkity + the latest 11ty into your project repo like so:
 
+> Slinkity requires Node 14+. So check `node -v` shows a major version of 14 or higher before installing slinkity.
```
Comment (ajcwebdev) [inline README.md:26]: Slightly cleaner way to say this:

> Slinkity requires Node 14+. You can check whether you are on a major version of Node 14 by running `node -v` in your terminal before installing Slinkity.

General stylistic question for, @Holben888, do we want to always capitalize Slinkity?

File: docs/src/docs/quick-start.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ npm init -y
 
 Then, install Slinkity + the latest 11ty into your project:
 
+> Slinkity requires Node 14+. So check `node -v` shows a major version of 14 or higher before installing slinkity.
```
Comment (ajcwebdev) [inline docs/src/docs/quick-start.md:17]: See previous note.

**Comment by ajcwebdev** [comment]
Thanks for this @AndrewLeedham! Made a small note about wording, but aside from that this looks good to me.

File: README.md (hunk 1 above)
Comment (AndrewLeedham) [reply README.md:26]: Perhaps adding at least as well? Or are we wanting to say this only works with Node 14?

> Slinkity requires Node 14+. You can check whether you are on at least a major version of Node 14 by running node -v in your terminal before installing Slinkity.

The logo and first sentence of the readme used Slinkity in title case. But agree would we should make sure that is the plan going forward 🙂

Comment (ajcwebdev) [reply README.md:26]: Yeah that's definitely something that might cause some confusion. Are we saying you need to be on 14+ and no higher (as in you shouldn't be on 15 or 16), or just need to be on something higher than 14. I think right now we want users to be on 14 and supporting 16 will require some further tests, but I'll have to double check with @Holben888.

Comment (ajcwebdev) [reply README.md:26]: Okay sounds like Node 16 should work.

Comment (Holben888) [reply README.md:26]: @AndrewLeedham @ajcwebdev  I'd say 14 and up to cover our bases! For what it's worth, I got a sample project building and running find on the latest Node (v16.13). Happy to change the wording for clarity:

> Slinkity requires Node v14 and up. You can check your versioning by running `node -v` in your terminal before trying Slinkity.

File: .nvmrc [hunk 3]
```diff
@@ -0,0 +1 @@
+14
```
Comment (Holben888) [inline .nvmrc:1]: Nice touch!

File: README.md (hunk 1 above)
Comment (AndrewLeedham) [reply README.md]: Only just realised I didn't capitalise the second occurrence 😅 Good catch @ajcwebdev.

**Review by ajcwebdev — APPROVED** [review]
Looks good to me, only other thing we may want to add would be a link to some kind of Node version manager like [nvm](https://github.com/nvm-sh/nvm), [volta](https://volta.sh/), or [fnm](https://github.com/Schniz/fnm). I know this is an opinionated matter though so we may want to leave this up to the developer instead of implicitly giving official blessing to one of these solutions. (I use Volta for what it's worth).

**Review by Holben888 — APPROVED** [review]
Love it! You're now a proud Slinkity contributor 😁

**Comment by Holben888** [comment]
> Looks good to me, only other thing we may want to add would be a link to some kind of Node version manager like [nvm](https://github.com/nvm-sh/nvm), [volta](https://volta.sh/), or [fnm](https://github.com/Schniz/fnm). I know this is an opinionated matter though so we may want to leave this up to the developer instead of implicitly giving official blessing to one of these solutions. (I use Volta for what it's worth).

Agreed. I'm personally loving Volta for my own purposes. Think I'll slap a [Volta pin](https://docs.volta.sh/reference/pin) into our `package.json` as well, separate from this PR. I'm fine using both an `.nvmrc` and a Volta config 👍

**Comment by Holben888** [comment]
Resolves #70

**Pull request merged**